style.css

```diff
@@ -0,0 +1,101 @@
+#general-fonts {
+    font-family: Arial, sans-serif;
+}
+
+#general-wrapper {
+    width: 800px;
+    height: 634px;
+    background-color: #F2F2F2;
+}
```

Comment:
why you use id for this general styles? for font family for example you could set it on * {} or something like that